run.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'app'
+
+class Run
+  def inicialize()
+    @app = App.new
+  end
+
+  def show_options()
+    puts ''
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def entry_point
+    puts 'Welcome to School Library App!'
+
+    loop do
+      show_options
+      app_choice = gets.chomp.to_i
+      options(app_choice)
```

Comment:
Nice job assigning the responsibility of input validation and request to this class😉. But let's also ensure every responsibility of requesting and validating inputs is done here to obey the single responsibility rule which states that every class should have a single responsibility. This makes our codebase maintainable and easier to understand, hence making it a display of professionalism👍🏼. currently, the functions of validating and requesting inputs are still present in the `app` class😉. let's do well to bring all of them here and call them when necessary in the respective classes. You can take a look at [this lesson ](https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design)for reference😉.

<img width="1015" alt="Screenshot 2023-07-31 at 17 08 07" src="https://github.com/Alejuda/oop-school-library/assets/64643995/3a281b6c-c794-419d-a832-d35969029902">